analysis/codelists.py

```diff
@@ -0,0 +1,145 @@
+from cohortextractor import (
+    codelist_from_csv,
+    codelist,
+)
+
+
+# OUTCOME CODELISTS
+covid_identification = codelist_from_csv(
+    "codelists/opensafely-covid-identification.csv",
+    system="icd10",
+    column="icd10_code",
+)
+
+covid_pos_primcare_code = codelist_from_csv(
```

Comment:
The current version of protocol says we will only do a sensitivity analysis in SGSS

Reply:
Ok, we will remove